testing/cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,298 @@
+/// <reference types="cypress" />
```

Comment:
Do we need actions.spec file?

Reply:
It's an example test file, for testing the framework. we can deleted or modify it later.